application/src/test/java/org/opentripplanner/routing/algorithm/mapping/__snapshots__/ElevationSnapshotTest.snap

```diff
@@ -116,14 +116,14 @@ org.opentripplanner.routing.algorithm.mapping.ElevationSnapshotTest.directBike=[
           "walkingBike" : false
         }
       ],
-      "startTime" : "2009-10-21T22:58:00.000+00:00",
+      "startTime" : "2009-10-21T22:57:59.000+00:00",
       "tooSloped" : false,
       "transfers" : 0,
       "transitTime" : 0,
       "waitingTime" : 0,
       "walkDistance" : 2636.74,
       "walkLimitExceeded" : false,
-      "walkTime" : 720
+      "walkTime" : 721
```

Comment:
I wonder if the walk time increases because of the `parentEdge.getInAngle()` -> `parentEdge.getOutAngle()` fix https://github.com/opentripplanner/OpenTripPlanner/blob/245247e3e7f03f62a10ed13174078b5df2619114/application/src/main/java/org/opentripplanner/street/model/edge/TemporaryPartialStreetEdge.java#L81-L88

Reply:
Yes. The turn duration for walking depends on the turn angle.